### PR TITLE
Add Issues Modal Styling

### PR DESCRIPTION
**Pull request by leikkisa**
connects #6984

### CODE REVIEW

**Comment by leikkisa** [comment]
Carola and I decided to make the buttons smaller when the screen is smaller, so they still all fit on one line.

Smaller buttons
screen shot 2018-10-17 at 11 54 39 am

Regular
screen shot 2018-10-17 at 11 55 03 am

File: client/app/styles/_intake.scss [hunk 1]
```diff
@@ -53,8 +53,17 @@
 }
 
 .intake-add-issues .cf-modal-body {
-  width: 770px;
-  overflow-y: auto;
+  width: 750px;
+  max-height: 95%;
+
+  @media only screen and (max-width: 760px) {
+    width: 650px;
```
Comment (pkarman) [inline client/app/styles/_intake.scss]: what happens when the screen width is less than 650px? I wonder if we should use `max-width: 95%` for the modal so it is always relative to the screen width?

Comment (leikkisa) [reply client/app/styles/_intake.scss]: I wasn't doing fully responsive design, just designing for the case where someone might use Caseflow on half of their screen.  Admittedly, I determined this number based on my Windows laptop resolution, which probably isn't the most thorough.  We got some feedback that the Queue research team designs for a common browser width of 768px.

If we use max-width, then the three modals won't have the same width (the ones with less wide content will be narrower), but we could try to do that for the smaller sizes.  The buttons will move at a smaller size, but it doesn't look that bad.  I can put in another case for smaller than 650px where that happens.

Comment (pkarman) [reply client/app/styles/_intake.scss]: cool. let's try it and see what feedback we get.

**Pull request merged**